fv3core/.jenkins/actions/run_regression_tests.sh

```diff
@@ -2,8 +2,17 @@
 set -e -x
 BACKEND=$1
 EXPNAME=$2
+SAVEPOINT=${3:-All}
+ORCHESTRATION=${4:-none}
 XML_REPORT="sequential_test_results.xml"
-export TEST_ARGS="-v -s -rsx --backend=${BACKEND} "
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
If we install `dace` as part of the base environment, you can avoid all of the changes here apart from the savepoint selection. We could avoid the savepoint selection as well if we instead used a hard-coded list of savepoint names in `conftest.py` that we want to test with dace, and updated those in our branch as new ones work. Has the added benefit of regression testing all dace-working code.